tgui/packages/tgui/interfaces/AirAlarm.js

```diff
@@ -113,10 +114,17 @@ const AirStatus = (props, context) => {
               <AnimatedNumber value={air.temperature} /> K / <AnimatedNumber value={air.temperature_c} /> C&nbsp;
               <Button
                 icon="thermometer-full"
-                content={air.temperature_c + " C"}
+                content={target_temp + " C"}
                 onClick={
                   () => act('temperature')
                 } />
+              <Button
+                content={air.thermostat_state ? "On" : "Off"}
+                selected={air.thermostat_state}
+                icon="power-off"
+                onClick={
+                  () => act('thermostat_state', { val: (air.thermostat_state === 1 ? 0 : 1) })
+                } />
```

Comment:
If you're doing the above, the argument here can also be removed.
```suggestion
                onClick={
                  () => act('thermostat_state')
                } />
```